File: Solutions/PAT/Basic/1002.lua
#!/usr/bin/env lua
-- 写出这个数 (20)

words = {
  'yi', 'er', 'san', 'si', 'wu',
  'liu', 'qi', 'ba', 'jiu'
}
words[0] = 'ling'
sum = 0
number = io.read('*line')
for i=1, #number do
  sum = sum + tonumber(number:sub(i, i))
end
sum = tostring(sum)
for i = 1, #sum do
  if i ~= 1 then io.write(' ') end
  io.write(words[tonumber(sum:sub(i, i))])
end

File: Solutions/PAT/Basic/1002.py
#!/usr/bin/env python3
# 写出这个数 (20)

from sys import stdout

words = [
  'ling', 'yi', 'er', 'san', 'si',
  'wu', 'liu', 'qi', 'ba', 'jiu',
]
s = sum([int(i) for i in input()])
for idx, i in enumerate(str(s)):
  if idx != 0:
    stdout.write(' ')
  stdout.write(words[int(i)])